### ISSUE TITLE
您好，这还是MPSR的代码吧？

### ISSUE THREAD

**Issue by LiangYang666** [AmingWu/Universal-Prototype-Enhancing#3]

**Comment by AmingWu** [comment]
Yes, the current code is based on MPSR. But our method could be easily plugged into other methods. We will continue to update the code. Thank you very much.
